tools/pileometh/test-data/cg100.fa.fai

```diff
@@ -0,0 +1,837 @@
+
```

Comment:
This file looks strange and it not a `fai` file. It looks like you tried to download it from github?